gemfile.d/foreman_bootdisk.rb

```diff
@@ -0,0 +1,6 @@
+# this gem is required by foreman_fog_proxmox
+gem 'simplecov'
+
+# For testing the foreman bootdisk proxmox extensions
+# this gem file is required
+gem 'foreman_fog_proxmox', '>= 0.13.4'
```

Comment:
Can you name this file `proxmox.rb` or something similar? So it is clear in the packaging phase what does this carries. Thanks for extracting this into separate file, in Satellite we would not distribute this one. By the way, I do not see this file included from the Gemfile, I think you might want to add a code that includes it.

@stejskalleos this file must not be distributed in Satellite package as Proxmox is not supported or shipped in Satellite repos, make sure there is a BZ prior merging (or preferably a patch)

@theforeman/packaging is this an acceptable solution? do we have a SPEC that would tell us if a package is built for upstream or downstream?

Reply:
Of course not. That would make it even more complicated than it already is.

If that's a test only dependency (aka: otherwise the plugin dymamically detects the presence, or absence of f_f_proxmox), it should not be in this file at all. 

We don't want to force upstream users of bootdisk to install proxmox, don't we?

Reply:
Man I completely forgot about poor upstream users. You are right, this enforces them to install proxmox, thus making a packaging dependency.

I am not sure how to solve this plugin to plugin dependency, do we have a precedent maybe?

Reply:
Yes this is test dependency. The tests were failing if I don't add this dependency. Do you any better way to handle it?

Reply:
One idea I have is to gate this behind an ENV variable, by default the dependency would not be required thus making all those tests to be skipped. So user who is not interested in testing Proxmox would just not notice it. But in your workflow, you can set the variable and test everything.

Reply:
```
group :test do
  gem 'blah'
end
```

wouldn't that work?

Reply:
This works.

Reply:
Nah, this skipped all proxmox tests.

Reply:
Yeah, it needs different approach.
I didn't quite understand ENV variable method. @lzap Can you please explain it?

Reply:
Well, now that I am thinking about it, bundler should pull the dependency automatically and since I haven't realized this is a test only dependency, I think the solution suggested by Evgeni is okay. So don't introduce any variables.